src/kirby/scss/base/_functions.scss

```
@@ -44,6 +44,11 @@
   }
 }
 
+@function get-text-color($variant) {
+  $value: _get-map-prop($text-colors, $variant);
+  @return unquote('var(--kirby-text-' + $variant + ')');
```

Include `color` in variable name:
```suggestion
  @return unquote('var(--kirby-text-color-' + $variant + ')');
```